endpoints/getting-started/client/main.go

```diff
@@ -100,8 +100,10 @@ func generateJWT(saKeyfile, saEmail, audience string, expiryLength int64) (strin
 // [START endpoints_jwt_request]
 
 // makeJWTRequest sends an authorized request to your deployed endpoint.
-func makeJWTRequest(signedJWT, url string) (string, error) {
-	client := &http.Client{}
+func makeJWTRequest(signedJWT, url string, timeout time.Duration) (string, error) {
+	client := &http.Client{
+		Timeout: timeout,
```

Comment:
Not sure this needs to be an argument to the function - `10 * time.Second` is probably good. Up to you.

Reply:
yeah, that sounds easier